mlir/tools/rocmlir-driver/rocmlir-driver.cpp

```diff
@@ -370,9 +370,9 @@ int main(int argc, char **argv) {
 #endif
   MLIRContext context(registry);
   context
-      .loadDialect<xmodel::XModelDialect, rock::RockDialect, func::FuncDialect,
```

Comment:
Did you mean to get rid of the `xmodel::` load?

Reply:
Like, given that CI passed, it's probably fine and we can merge, but I thought I'd check

Reply:
good catch; a rebasing artifact.

Reply:
I've changed it now.. 

Putting on a TDD hat, I would agree with you but I should not be doing that in this PR :).